src/skin/legacy/tooltips.cpp

```diff
@@ -391,13 +391,23 @@ void Tooltips::addStandardTooltips() {
             << tr("BPM Tap")
             << tr("When tapped repeatedly, adjusts the BPM to match the tapped BPM.");
 
+    add("beats_set_marker")
+            << tr("Set Beat Marker")
```

Comment:
It his actually "Set Bar" Or "Set Down-Beat"?

Reply:
Both, but that also doesn't quite fit either. I'm still contemplating if another name would be better, e.g. "Set Tempo Region Marker" or something like that.

Reply:
What are the cases where this is not a downbeat? 

I can imagine if one does not bother with downbeats. We may consider to express that as beatsPerBar = 0.

Do you have something else In mind?

Reply:
I like "Add Tempo Marker".  "set" implies there is only one, and if I'm not mistaken we can have more than one of these, right?

Reply:
Yes, there can be multiple tempo markers. The distance between 2 markers needs to be at least 1 bar (= 4 beats currently). If you try to place a tempo marker very close to another on (within a range of 2 beats), the editing UI is smart enough to move the nearby marker instead of inserting one (which would double the BPM between the 2 markers, which is very likely undesired). So "Add" would be misleading IMHO.